lib/verbalize/build.rb

```diff
@@ -34,18 +35,19 @@ def initialize(#{declaration_arguments_string})
       CODE
     end
 
-    attr_reader :required_keywords, :optional_keywords
+    attr_reader :required_keywords, :optional_keywords, :default_keywords
 
     private
 
     def all_keywords
-      required_keywords + optional_keywords
+      required_keywords + optional_keywords + default_keywords
     end
 
     def declaration_arguments_string
-      required_segments = required_keywords.map { |keyword| "#{keyword}:" }
-      optional_segments = optional_keywords.map { |keyword| "#{keyword}: nil" }
-      (required_segments + optional_segments).join(', ')
+      required_segments  = required_keywords.map { |keyword| "#{keyword}:" }
+      optional_segments  = optional_keywords.map { |keyword| "#{keyword}: nil" }
+      default_segments   = default_keywords.map  { |keyword| "#{keyword}: self.defaults[:#{keyword}].call" }
```

Comment:
... here, which is why we need to segregate them out inside of `Verbalize::Action`.